src/config/constants/types.ts

```diff
@@ -153,3 +154,26 @@ export type PageMeta = {
   description?: string
   image?: string
 }
+
+export enum LotteryStatus {
+  PENDING = 'pending',
+  OPEN = 'open',
+  CLOSE = 'close',
+  CLAIMABLE = 'claimable',
+}
+
+export interface LotteryTicket {
+  id: string
+  number: string
+  status?: boolean
+  rewardBracket?: number
+  roundId?: string
+  cakeReward?: SerializedBigNumber
```

Comment:
Same here, are these really not required?

Reply:
These are attached as part of rewards-fetching, not all `LotteryTicket`s have them and they aren't required